recipes-bsp/u-boot/u-boot-fslc/colibri-imx6/fw_env.config

```diff
@@ -4,4 +4,4 @@
 # block which occupies the last sector (hence -0x200)
 
 # Block device name	Device offset	Env. size
-/dev/mmcblk0boot0	-0x2200		0x2000
+/dev/mmcblk1boot0	-0x2200		0x2000
```

Comment:
Commit log:

colibri-imx6: u-boot-fslc: Fix block device name in fw_env.config